### PR TITLE
contrib specification directory for vendor specs

### LINKED ISSUE #1110: Contrib directory for vendor specific specifications
**What are you trying to achieve?**

A directory of vendor specific specifications for components like propagators, processors, resources, etc, to be where vendors must submit their contributions in addition to any implementations to language specific repos.

### PR DESCRIPTION
**Pull request by tsloughter**
Fixes #1110

### CODE REVIEW

File: contrib/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Vendor Specific Contributions
+
+This folder will be used to include vendor specific specifications for
+components, such as propagators, processors, resources, exporters, etc
```
Comment (jkwatson) [inline contrib/README.md:4]:
```suggestion
components, such as propagators, processors, resources, exporters, etc.
```

**Review by jkwatson — APPROVED** [review]

File: contrib/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Vendor Specific Contributions
+
+This folder will be used to include vendor specific specifications for
+components, such as propagators, processors, resources, exporters, etc.
```
Comment (SergeyKanzhelev) [inline contrib/README.md:4]: let's add a note on support statement like this: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/#support

**Review by SergeyKanzhelev — CHANGES_REQUESTED** [review]
let's add clarification on the folder supportability. Like I suggested in the comment.

**Comment by SergeyKanzhelev** [comment]
I think adding the comment about supportability like this https://github.com/open-telemetry/opentelemetry-dotnet-contrib/#support will address today's meeting concerns. But I'd let @reyang, @tedsuo, and @alolita to comment here

**Comment by alolita** [comment]
@SergeyKanzhelev As we discussed in the maintainers meeting - I am working on a proposal for hosting vendor contribs. Will discuss w @tedsuo and link the issue here as soon as I file it. Thanks for sharing the .net guidelines by @reyang

**Comment by SergeyKanzhelev** [comment]
@tsloughter I'll close this PR for now than unless the decision is made. Thank you for filing PR!

**Pull request closed without merging**